### PR TITLE
Bug fixes and updates

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: translators/object.py [hunk 1]
```diff
@@ -118,6 +120,13 @@ def flush_entities(self, assembly):
     def update_transform(self, time, matrix):
         self.__ass_inst.transform_sequence().set_transform(time, self._convert_matrix(matrix))
 
+    def _convert_matrix(self, m):
```
Comment (dictoon) [inline translators/object.py]: Private methods must start with two underscores.

File: translators/object.py [hunk 2]
```diff
@@ -161,3 +170,9 @@ def flush_entities(self, assembly):
 
     def update_transform(self, time, matrix):
         self.__ass_inst.transform_sequence().set_transform(time, self._convert_matrix(matrix))
+
+    def _convert_matrix(self, m):
```
Comment (dictoon) [inline translators/object.py]: Same here.

File: translators/object.py (hunk 1 above)
Comment (dictoon) [reply translators/object.py]: Do I understand correctly that this method not only returns a new matrix, but also modifies the argument that is passed to it? Is this modification visible to the outside? I.e.

	m1 = some matrix
	m2 = self._convert_matrix(m1)
	# is m1 different now?

Comment (jdent02) [reply translators/object.py]: It’s not private.  It’s an override for the same function in the base Translator class

Comment (jdent02) [reply translators/object.py]: It’s an override for the same function in the base Translator class.  Esteban said those have single underscores. Only internal functions which are not inherited need double underscores.  This convention is followed throughout blenderseed.

Comment (dictoon) [reply translators/object.py]: Thanks for the explanation. Today I learn!

Comment (jdent02) [reply translators/object.py]: I think the single underscore is the closest thing Python has to a 'protected' member.

**Pull request merged**